sdk/cosmos/azure-spring-data-2-2-cosmos/README.md

```diff
@@ -1,9 +1,73 @@
 # Azure Spring Data 2.2 Cosmos client library for Java
 
-## Key concepts
+**Azure Spring Data Cosmos** provides Spring Data 2.2.x support for Azure Cosmos DB using the [SQL API][sql_api_query], based on Spring Data framework. 
+[Azure Cosmos DB][cosmos_introduction] is a globally-distributed database service which allows developers to work with data using a variety of standard APIs, such as SQL, MongoDB, Cassandra, Graph, and Table.
+
+[Source code][source_code] | [Package (Maven)][azure_spring_data_2_2_cosmos_maven] | [API reference documentation][api_documentation] | [Product documentation][azure_spring_data_cosmos_docs] |
+[Samples][samples]
+
+## Spring data version support
+This project supports [spring-data-commons 2.2.x][azure_spring_data_2_2_cosmos] version.
+
 ## Getting started
+### Include the package
+If you are using Maven, add the following dependency. 
+
+[//]: # ({x-version-update-start;com.azure:azure-spring-data-2-2-cosmos;current})
+```xml
+<dependency>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-spring-data-2-2-cosmos</artifactId>
+    <version>3.0.0-beta.1</version>
+</dependency>
+```
+[//]: # ({x-version-update-end})
+
 ## Key concepts
-## Examples
+Please refer to [Azure Spring Data Cosmos Core module][azure_spring_data_cosmos_core_readme_key_concepts] for key concepts. 
+
 ## Troubleshooting
+Please refer to [Azure Spring Data Cosmos Core module][azure_spring_data_cosmos_core_readme_troubleshooting] for troubleshooting guide.
+
+## Examples
+- Please refer to [sample project here][samples].
+
 ## Next steps
+- Read more about azure spring data cosmos [here][azure_spring_data_cosmos_docs].
+- [Read more about Azure CosmosDB Service][cosmos_docs]
```

Comment:
- Read more about [azure spring data cosmos][azure_spring_data_cosmos_docs].
- Read more about [Azure CosmosDB Service][cosmos_docs]

Reply:
Thanks, fixed it.